package.json

```diff
@@ -67,18 +72,18 @@
     "eslint-plugin-react": "^7.10.0",
     "esm": "^3.0.65",
     "minimist": "^1.2.0",
-    "nyc": "^12.0.2",
+    "nyc": "^13.3.0",
     "opn": "^5.3.0",
     "prettier-eslint-cli": "^4.7.1",
     "tap-spec": "^5.0.0",
     "tape": "^4.9.1",
     "tape-promise": "^3.0.0",
-    "testem": "^2.8.2"
+    "testem": "^2.14.0",
+    "vscode": "^1.1.33"
```

Comment:
try moving `vscode` under `peerDependencies` instead to avoid linting error.

Reply:
`vscode` doesn't seem to work as a peer dependency. The `npm install` fails. I've reviewed some official MS samples, and the dev dependency seems to be the place to put it ([example](https://github.com/Microsoft/vscode-extension-samples/blob/master/basic-multi-root-sample/package.json)). When it's in the primary dependencies, `vsce` refuses to compile it and says it should be moved to `devDependencies`.